src/geometry/losses.jl

```diff
@@ -0,0 +1,6 @@
+function image_mse(y, ŷ)
+    l = map((x, y) -> (x - y), y, ŷ)
+    l = mapreducec.(x->x^2, +, 0, l)
+    l = sum(l)
+    l
+end
```

Comment:
I think this is some ad-hoc script for test and should be moved to `test/`.

Reply:
I think I should keep it here. I was going to build some loss functions for Images anyways, and although this is a bit messy, I can build upon this only. I had also written tests for this, so I will keep it here for now.

Reply:
> I was going to build some loss functions for Images anyways, and although this is a bit messy, I can build upon this only.

If we want more generic support, we should consider improve ImageDistances and not writing ad-hoc scripts like this. The role of DiffImages are very clear to me; it is a playground to bring AD support to some operations in JuliaImages. Anything, if possible, should be moved to upstream packages. If DiffImages wants to become another ecosystem, I don't think we have enough people willing to contribute and maintain.

Reply:
This sounds more practical. I think using ImageDistances is much better than keeping all the things in DiffImages. I shall make the changes you have instructed.